ceph/ceph_admin/__init__.py

```diff
@@ -88,12 +88,13 @@ def distribute_cephadm_gen_pub_key(self, nodes=None):
 
     def set_tool_repo(self):
         """Add the given repo on every node part of the cluster."""
-        base_url = self.config.get("base_url")
-        if not base_url.endswith("/"):
-            base_url += "/"
-        cmd = f"yum-config-manager --add-repo {base_url}compose/Tools/x86_64/os/"
-        for node in self.cluster.get_nodes():
-            node.exec_command(sudo=True, cmd=cmd)
+        if not self.config.get("hotfix_repo"):
+            base_url = self.config.get("base_url")
```

Comment:
Would you be able to modify this to `base_url = self.config['base_url']`? `base_url` is a required parameter for building out the cluster... the change would help in failing fast.

Reply:
Ok, I've made that change.